services/py-api/src/server/routes/participants_routes.py

```diff
@@ -42,7 +42,7 @@ def _handler(service: ParticipantService = Depends(_service)) -> ParticipantHand
 
 
 @participants_router.post(
-    "", responses={201: {"model": ParticipantRegisteredInTeamResponse}, 409: {"model": ErrResponse}}
+    "", status_code=201, responses={201: {"model": ParticipantRegisteredInTeamResponse}, 409: {"model": ErrResponse}}
```

Comment:
I added this to tell the Swagger docs that this is the default status code to be returned. otherwise, it puts 200 by default. and we get possible returned response codes: 200, 201, 409, 422 and we want only 201, 409, 422

Reply:
Nice!